index.html

```diff
@@ -14,46 +14,190 @@
       rel="stylesheet"
     />
 
-    <title>portfolio-setup Project</title>
+    <title>portfolio-setup</title>
   </head>
   <body>
-    <!-- my-logo section -->
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
```

Comment:
Nice work once again, it would be nice if you fulfil the requirement of using `display: grid` for the main container. You can also use `justify-items: center` to make the portfolio align to the center.
So in your CSS file, your code will look like this:

```
.middle-section {
display: grid;
justify-items: center;
}